qualtran/bloqs/arithmetic/addition.py

```diff
@@ -62,30 +62,43 @@ class Add(Bloq):
     Implements $U|a\rangle|b\rangle \rightarrow |a\rangle|a+b\rangle$ using $4n - 4 T$ gates.
 
     Args:
-        bitsize: Number of bits used to represent each integer. Must be large
+        dtype: Quantum datatype used to represent each integer. Must be large
```

Comment:
this docstring needs modification. the class now has two Args `a_dtype` and `b_dtype` not a single `dtype`